### ISSUE TITLE
Deprecation warning seen

### ISSUE THREAD

**Issue by JohnVillalovos** [morganfainberg/positional#24]
I am seeing a deprecation warning being generated:

```
python3.5/site-packages/positional/__init__.py:74: DeprecationWarning: inspect.getargspec() is deprecated, use inspect.signature() instead
  spec = inspect.getargspec(func)
```

**Comment by morganfainberg** [comment]
Fixed by #25

**Comment by morganfainberg** [comment]
I'll do a release once I'm not in an airport and having issues with ssh connections.
